backend/src/main/java/com/hyooun/wilson/review/adapter/out/persistence/ReviewMapper.java

```diff
@@ -0,0 +1,33 @@
+package com.hyooun.wilson.review.adapter.out.persistence;
+
+import com.hyooun.wilson.project.domain.Project;
+import com.hyooun.wilson.review.domain.Review;
+import org.springframework.stereotype.Component;
+
```

Comment:
ReviewMapper 클래스에 Review 객체를 매핑하는 메서드를 구현해 주세요.

```suggestion
Review mapToDomainEntity(ReviewJpaEntity reviewJpaEntity) { ... }
```